pkg/cmd/cli/describe/printer_test.go

```diff
@@ -33,6 +34,7 @@ var MissingPrinterCoverageExceptions = []reflect.Type{
 	reflect.TypeOf(&authorizationapi.SubjectAccessReview{}),
 	reflect.TypeOf(&authorizationapi.ResourceAccessReview{}),
 	reflect.TypeOf(&deployapi.DeploymentConfigRollback{}),
+	reflect.TypeOf(&deployapi.DeploymentLogOptions{}),
```

Comment:
You can't add this here.  If there's not going to be a printer, add it above with a justification.  Since we don't take a `-f` for `oc get`, it's probably good enough to say that we don't ever return this object from the API.